designdocs/logical_batching.md

```diff
@@ -0,0 +1,101 @@
+# Summary
+
+In other documents, we explore ways to structure an explicit batch of messages.
+That is, in terms of concrete structure. This document has a different aim --
+deriving the idea of a batch logically, as a product of fields present on other
+entities in our system.
+
+## Background
+
+We are currently experimenting with a new system architecture that has the
+following entities:
+
+1. Schema
+   - id: u16
+   - format: Format
+   - format_type: FormatType
```

Comment:
Reminder to modify this to match whatever the decision is for what this will end up being called. I think we said `Model` and `ModelType` yesterday.